app/scripts/contentscript.js

```diff
@@ -100,8 +101,8 @@ const TIME_45_MIN_IN_MS = 45 * 60 * 1000;
  * The keep-alive logic should not work for non-dapp pages.
  */
 const IGNORE_INIT_METHODS_FOR_KEEP_ALIVE = [
-  MESSAGE_TYPE.GET_PROVIDER_STATE,
```

Comment:
We have to be cautious not to replace valid usages of `MESSAGE_TYPE` which this appears to be as it's concerning JSON-RPC methods.

Reply:
Reverted all unnecessary changes (RPCs and metrics)